docs/browser-engine/webview2.md

```diff
@@ -0,0 +1,72 @@
+---
+title: WebView2
+---
+
+# WebView2
+
+<!-- ✍ Please populate the content `/data/technologies/webview2.json`
+first. The text starting with `{{` will be interpolated in build time with
+the information in that file. -->
+
+WebView2 is one of the most popular technologies to build hybrid applications that leverage the best of both the native and web! With WebView2 you can embed html, css, and javascript to modernize your applications. This enables your app to have native app benefits like access to file manager and custom drm systems while getting the reach of the web. 
+
+WebView2 has recently GA'd but has been widely adopted by apps like Office, Outlook, Morgan Stanley, and many others. Watch how Morgan Stanley uses WebView2 [here](https://www.youtube.com/watch?v=8y3ZCzw3LtA).
+
+
+<!-- ✍ Please write about the technology, when it was created, what type of technology is it,
+some of the key characteristics for developers, etc. -->
+
+Althought the traditional IE WebBrowser control dates back to (2008 -- #JS can we find the actual date), WebView2 is a newer technology that has modernized the embeddable web browser by adopting the microsoft edge chromium platform. WebView2 has recently GA'd on Win32 C/C++, .NET, and expects to have UWP support soon. 
+
+WebView2 has both an evergreen and fixed distribution mode. With the default Evergreen mode, WebView2 will get the latest features, security, and bug-fixes automatically! For developers who want more control over their app updates we also offer the fixed version which allows you to ship webview2 with your app.
+
+WebView2 is backed by Microsoft, and continues to engage with the developer community and take feature requests and bug fixes.
+
+## Release cadence, version support and update model
+
+<!-- ✍ How often is there a new version? Is it predictable? Does it use an evergreen model (i.e.:
+it gets updated automatically) or do developers need to take care of that? Does it depend on other
+projects to ship? -->
+
+There are 2 parts to WebView2: the SDK, which comes in the form of a NuGet package, and the WebView2 Runtime, which take and optimize the edge browser binaries. The SDK and Runtime get updated every 4 weeks, in alignment with Edge. WebView2 offers a prerelease SDK with up and coming experimental features and a stable release SDK. 
```

Comment:
I'd stick with the 6 release cadence and make [a note similar to the one in Electron](https://github.com/crossplatform-dev/crossplatform.dev/pull/6/files#diff-17c6f8d1d1ab64c44676e2301283cfcb698ac6ff80bed1defb5b9140c1395256R33) about that switching soon-ish.

Reply:
Would it be possible to add links to the NuGet package and the Runtime?

Also how is the versioning supposed to work? Is there only one major version that gets updated every few weeks and supposedly no breaking changes? Are there minor updates between those 6 weeks periods?

We might want to move the distribution modes here (or mention them again) as it is part of the update process.